### ISSUE TITLE
No module named 'distances'

### ISSUE THREAD

**Issue by ashleyblackmore** [pymc-devs/pymc3#528]
```
    from pymc import (Exponential,
  File "/home/user/Bauen/bayes/env/lib/python3.3/site-packages/pymc/__init__.py", line 47, in <module>
    from . import gp
  File "/home/user/Bauen/bayes/env/lib/python3.3/site-packages/pymc/gp/__init__.py", line 27, in <module>
    from .cov_funs import *
  File "/home/user/Bauen/bayes/env/lib/python3.3/site-packages/pymc/gp/cov_funs/__init__.py", line 2, in <module>
    from .cov_utils import *
  File "/home/user/Bauen/bayes/env/lib/python3.3/site-packages/pymc/gp/cov_funs/cov_utils.py", line 4, in <module>
    from . import wrapped_distances
  File "/home/user/Bauen/bayes/env/lib/python3.3/site-packages/pymc/gp/cov_funs/wrapped_distances.py", line 5, in <module>
    import distances
ImportError: No module named 'distances'
```

I believe I compiled with gfortran 4.8:

```
01:30 $ gfortran -v
Using built-in specs.
COLLECT_GCC=gfortran
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-linux-gnu/4.8/lto-wrapper
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Ubuntu/Linaro 4.8.1-10ubuntu9' --with-bugurl=file:///usr/share/doc/gcc-4.8/README.Bugs --enable-languages=c,c++,java,go,d,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.8 --enable-shared --enable-linker-build-id --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.8 --libdir=/usr/lib --enable-nls --with-sysroot=/ --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --enable-gnu-unique-object --enable-plugin --with-system-zlib --disable-browser-plugin --enable-java-awt=gtk --enable-gtk-cairo --with-java-home=/usr/lib/jvm/java-1.5.0-gcj-4.8-amd64/jre --enable-java-home --with-jvm-root-dir=/usr/lib/jvm/java-1.5.0-gcj-4.8-amd64 --with-jvm-jar-dir=/usr/lib/jvm-exports/java-1.5.0-gcj-4.8-amd64 --with-arch-directory=amd64 --with-ecj-jar=/usr/share/java/eclipse-ecj.jar --enable-objc-gc --enable-multiarch --disable-werror --with-arch-32=i686 --with-abi=m64 --with-multilib-list=m32,m64,mx32 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
Thread model: posix
gcc version 4.8.1 (Ubuntu/Linaro 4.8.1-10ubuntu9) 
```

**Comment by kyleam** [comment]
I think this was fixed with #492 (48ff47d).

**Comment by ashleyblackmore** [comment]
Correct! Thank you.
